gen/MotController_data.yml

```diff
@@ -185,7 +185,7 @@ c_MotController_SetDemand:
 
     calc_modes = [
         ControlMode.MotionMagic, ControlMode.Position,
-        ControlMode.Velocity, ControlMode.Current]
+        ControlMode.Velocity, ControlMode.Current, ControlMode.PercentOutput]
```

Comment:
Ehhhh. Could you please make this conditional on the ramp rate config instead?

Reply:
What if the ramp rate is set after the demand is set? Bit of an extreme edge case, as set() is usually called every loop anyways.

Reply:
Then the demand gets to the motor controller before the config does?

Reply:
Hm.

We do need to think about a better way to do all of this, and in particular we need to prevent it from killing so much CPU as it currently does. This isn't related to this PR per se, but it is an issue that needs to be addressed.

I wonder if it would be more efficient to get rid of the notifier and just compute multiple iterations on the main physics loop? Though, I suppose that only works well assuming that most processing happens on a single thread.